app/lib/three_scale/filter_arguments.rb

```diff
@@ -14,9 +14,18 @@ def initialize(arguments = [])
     end
 
     def filter
-      return FILTER.filter(@arguments) if @arguments.is_a?(Hash)
+      return FILTER.filter(@arguments) if parameters_arguments?
 
       @arguments.map { |argument| argument.is_a?(Enumerable) ? FILTER.filter(argument) : argument }
     end
+
+    def parameters_arguments?
+      case @arguments
```

Comment:
1-liner alternative:

```ruby
(@arguments.class.ancestors & [ActionController::Parameters, Hash]).present?
```

Reply:
👍 nice one but might be complicated to read